.github/workflows/detect-conflicting-prs.yml

```diff
@@ -0,0 +1,71 @@
+name: "Detect conflicting PRs"
+
+on:
+  workflow_dispatch:              # manually
+  # So that PRs touching the same files as the push are updated
+  push:
+  # So that the `dirtyLabel` is removed if conflicts are resolved
+  pull_request_target:            # - A pull request (even with conflicts)
+    types:
+      - synchronize               #   pushing more commits
+
+permissions:
+  # no checkouts/branching needed
+  contents: none
+  # need by "eps1lon/actions-label-merge-conflict" to manage PR label/comments
+  pull-requests: write
+
+# This allows a subsequently queued workflow run to interrupt/wait for previous runs
+concurrency:
+  group: '${{ github.workflow }}'
+  cancel-in-progress: false  # true: interrupt, false = wait for
+
+jobs:
+  detect-prs:
+    name: Detect
+    if: ${{ github.actor != 'dependabot[bot]' }} # avoid dependabot PRs
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Label conflicting PRs that are open
+        id: pr-labeler
+        uses: eps1lon/actions-label-merge-conflict@v2.0.1
+        with:
+          repoToken:  ${{ secrets.GITHUB_TOKEN }}
+          retryAfter: 30 # seconds
+          retryMax:   5  # atemps
+          dirtyLabel: conflicts
+          commentOnDirty: |
+            Oh no 😟! A dirty status was reached.
```

Comment:
I think we should change this sentence, maybe to "Oh no! Conflicts have been found."

Reply:
Done! e5683a6334942a01eb5b7f8a0208fc83d8229c02